packages/widget_toolkit_pin/test/mocks/pin_code_service_mock.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+
+class PinCodeServiceMock implements PinCodeService {
```

Comment:
What's the reason for creating a mock class instead of using the mockito package ?